### PR TITLE
Endo Daemon CLI Subcommand Descriptions

### PR DESCRIPTION
**Pull request by kriskowal**
This change adds descriptions to all the `endo` CLI subcommands so far implemented.

### CODE REVIEW

File: packages/cli/src/endo.js [hunk 1]
```diff
@@ -324,80 +380,95 @@ export const main = async rawArgs => {
...
-      const pet = await E(bootstrap).provide(name);
-      console.log(pet);
-    } catch (error) {
-      console.error(error);
-      cancel(error);
-    }
-  });
+  program
+    .command('show <name>')
+    .description('prints a representation of the named value')
```
Comment (dckc) [inline packages/cli/src/endo.js:403]: what does "representation" add?

```suggestion
    .description('prints the named value')
```

I tried the [console.log docs](https://nodejs.org/api/console.html#consolelogdata-args); they're no help: "Prints to stdout with newline. ... the arguments are all passed to `util.format()`)"

Another angle: "carries out the named formula and prints the result".

I see `provide` in the implementation, which suggests something like: "looks the name up in a cache; carries out the named formula on cache miss; prints the result". The relationship between cache entry lifetimes and the daemon lifetime also seems relevant.

File: packages/cli/src/endo.js [hunk 2]
```diff
@@ -324,80 +380,95 @@ export const main = async rawArgs => {
...
-    try {
-      const bootstrap = getBootstrap();
-      const iterable = await E(bootstrap).provide(name);
-      const iterator = await E(iterable)[Symbol.asyncIterator]();
-      for await (const iterand of makeRefIterator(iterator)) {
-        console.log(iterand);
+  program
+    .command('follow <name>')
+    .description(
+      'prints a representation of each value from the named async iterable as it arrives',
```
Comment (dckc) [inline packages/cli/src/endo.js:423]:
```suggestion
      'prints each value from the named async iterable as it arrives',
```

**Review by dckc — APPROVED** [review]
some of my comments are more relevant to a spec; I suppose command descriptions don't need to be a complete specification to be useful

**Comment by kriskowal** [comment]
Per this feedback, I took a run at making these descriptions much more concise at the end of the `endo` branch. The spike continues but I’ve left these PRs behind.

```
  open [options] <name> [filePath]                      opens a web page (weblet)
  run [options] [<file>] [<args>...]                    runs a program (runlet)
  make [options] [file]                                 make a plugin or a worker caplet (worklet)
  inbox [options]                                       read messages
  request [options] <informal-description>              ask someone for something
  resolve [options] <request-number> <resolution-name>  grant a request
  reject [options] <request-number> [message]           deny a request
  send [options] <party> <messag>                       send a message with @named-values @for-you:from-me
  adopt [options] <message-number> <name-in-message>    accept a @value from a message
  dismiss [options] <message-number>                    delete a message
  list [options]                                        show names
  remove [options] [names...]                           forget a named value
  rename [options] <from> <to>                          change the name for a value
  show [options] <name>                                 prints the named value
  follow [options] <name>                               subscribe to a stream of values
  cat [options] <name>                                  dumps a blob
  store [options] <path>                                stores a blob
  eval [options] <source> [names...]                    creates a value
  spawn [options] [names...]                            creates a worker
  bundle [options] <application-path>                   stores a program
  mkhost [options] <name>                               makes a separate mailbox and storage for you
  mkguest [options] <name>                              makes a mailbox and storage for a guest (peer or program)
  where                                                 prints paths for state, logs, caches, socket, pids
  start                                                 start the endo daemon
  stop                                                  stop the endo daemon
  restart                                               stop and start the daemon
  clean                                                 erases ephemeral state
  reset                                                 erases persistent state and restarts if running
  log [options]                                         writes out the daemon log, optionally following updates
  ping                                                  prints ok if the daemon is responsive
  help [command]                                        display help for command
```

**Pull request closed without merging**